services/frontend/src/scenes/ProjectDetails/ProjectDetailsLayout.js

```diff
@@ -184,4 +161,17 @@ const Changelog = ({sectionName}) => (
     </>
 );
 
+const ContributorBadge = ({name, role}) => {
+    return (
+        <div className="project-contributors-badge">
+            <Tooltip title={role} placement={"top"}>
```

Comment:
I suggest refactoring this whole 'Avatar' thing out into its component so that we can share it between the ProjectDetails page and the Profile page. 

Additionally, I'd suggest keeping the `Tooltip` as part of the Avatar, but just conditionally rendering it based on whether  or not a `tooltip` prop has been passed to the new `UserAvatar` component (or something along those lines).